General queries/WD AV Signature and Platofrm Version.txt

```diff
@@ -0,0 +1,15 @@
+//Author: António Vasconcelos
+//Twitter: https://twitter.com/anthonws
+// This query provides you the latest signature and platform (MoCamp) for Windows Defender AV
+FileCreationEvents
+| where InitiatingProcessCommandLine contains "MpSigStub.exe"
+| extend NewVersion=tostring(split(InitiatingProcessCommandLine, " ")[4]) 
+| summarize arg_max(NewVersion, *) by ComputerName
```

Comment:
why not by `EventTime`?
i.e. `arg_max(NewVersion, EventTime)`

Reply:
Agreed.